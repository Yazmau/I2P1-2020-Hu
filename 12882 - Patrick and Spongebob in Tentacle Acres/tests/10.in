84 80
55 57 77 69 16 80 8 26 19 51 87 92 45 43 8 96 16 65 93 20 88 66 69 84 8 28 40 17 94 48 91 89 84 86 37 93 16 12 71 57 93 11 1 44 43 52 86 82 67 96 13 88 84 13 6 7 79 63 29 74 73 24 28 15 92 20 48 73 61 53 76 57 17 60 25 33 55 10 31 12
67 72 94 69 51 76 66 27 69 22 28 86 86 68 59 36 58 54 77 98 17 66 15 9 15 25 99 77 73 68 13 74 23 69 42 97 84 55 36 19 76 97 74 3 9 52 45 91 6 4 15 58 47 11 8 45 40 6 60 18 98 99 98 88 32 61 24 70 19 95 89 91 6 73 78 67 63 48 45 97
96 3 37 31 19 97 59 90 91 100 43 75 8 93 73 42 71 86 89 87 92 6 59 40 94 32 27 97 31 45 23 32 49 98 92 53 6 63 89 38 4 26 100 20 42 96 38 73 57 79 39 36 84 6 12 93 65 16 84 38 33 46 59 83 84 53 74 46 98 33 5 91 8 12 7 22 73 83 54 88
93 20 76 94 77 70 75 46 97 25 90 68 81 63 78 66 17 24 38 8 21 51 51 61 82 86 88 19 6 55 36 47 4 1 95 44 78 91 53 31 68 53 39 93 36 61 95 13 25 23 76 62 82 30 52 18 94 5 40 25 10 14 36 75 96 91 8 30 16 48 72 17 97 62 81 11 93 71 18 13
10 23 72 48 54 85 90 36 12 72 86 80 32 98 18 79 59 41 96 12 37 5 62 31 33 55 55 74 60 49 64 73 2 91 99 98 20 32 99 14 76 19 31 46 36 22 7 99 12 59 30 11 79 38 10 17 86 52 94 72 57 71 35 49 64 20 36 45 8 43 87 4 63 22 81 81 75 83 65 11
79 65 50 49 32 7 33 67 50 94 12 55 29 65 69 93 81 4 23 12 26 35 20 82 32 90 74 48 20 95 14 76 29 68 96 20 4 73 41 74 76 15 36 98 19 96 37 58 96 77 41 31 38 18 92 93 27 18 65 31 31 76 94 32 49 55 81 36 49 11 60 89 29 93 78 34 11 83 88 7
20 94 68 54 5 94 54 24 29 78 6 7 44 13 55 1 27 54 84 19 49 97 93 32 71 3 86 57 86 71 4 64 56 7 86 18 70 27 70 67 89 82 54 18 36 23 63 67 23 49 25 79 83 59 93 11 84 86 37 57 98 94 27 53 61 13 11 10 9 67 53 31 67 27 15 86 1 84 65 8
2 77 48 99 52 50 70 55 87 17 14 53 29 94 13 11 93 7 77 86 63 24 97 98 15 100 47 95 81 6 69 10 75 73 26 43 42 5 2 90 77 16 21 54 10 42 40 5 11 37 30 75 2 5 92 46 33 21 74 59 52 40 100 84 78 13 1 36 23 48 52 50 24 43 62 58 74 94 34 43
51 31 69 83 24 43 22 67 38 42 82 48 44 31 45 18 35 42 40 41 66 64 72 87 86 78 70 59 95 67 82 77 89 9 62 100 89 84 77 35 49 56 93 45 55 73 48 5 18 46 8 14 72 31 75 95 98 82 80 16 63 43 92 68 70 16 25 53 19 94 43 96 2 78 86 46 68 8 61 7
64 7 14 71 35 3 4 44 77 2 68 2 33 100 72 99 25 47 47 43 48 52 11 73 14 60 37 24 47 47 57 3 24 84 91 25 77 36 63 85 73 54 42 83 38 58 23 38 22 75 67 45 31 10 25 74 8 18 75 29 21 7 91 62 47 62 6 51 17 8 24 55 84 55 11 73 12 79 40 58
82 58 65 19 20 1 29 79 56 69 99 63 57 50 59 57 95 95 8 97 87 9 76 64 69 8 62 47 41 45 11 89 19 22 86 49 99 60 16 53 38 19 77 18 67 38 53 73 6 45 67 99 100 50 50 26 9 49 43 14 45 99 52 77 47 78 4 99 29 39 77 59 54 95 61 40 91 74 29 69
43 32 65 49 35 95 74 84 98 78 42 16 74 1 28 61 97 18 57 98 77 66 84 89 29 14 7 7 81 95 96 61 6 14 29 32 24 68 32 94 66 75 80 63 42 8 62 22 98 7 81 73 11 78 78 80 42 58 12 34 14 35 73 18 52 37 24 55 7 22 25 9 44 73 36 26 75 70 10 54
81 76 2 48 64 9 10 86 17 13 67 5 25 96 49 75 100 16 28 79 77 24 12 24 93 35 10 50 56 53 1 27 50 81 18 31 95 2 12 37 87 53 84 17 95 73 2 32 50 71 63 34 89 66 85 99 7 29 78 33 6 21 61 76 65 78 28 99 80 14 84 81 63 74 56 72 82 79 2 69
86 42 92 14 71 5 40 38 12 32 61 41 11 88 32 94 6 23 52 3 39 98 14 52 97 10 62 80 28 65 32 91 5 54 99 94 67 99 42 94 95 6 38 69 66 16 41 23 26 21 78 15 76 83 51 72 33 67 50 96 100 22 20 44 78 30 13 2 78 38 46 78 41 49 41 58 67 76 25 65
76 95 39 76 60 84 38 96 23 4 32 57 22 9 3 58 37 58 97 31 47 1 64 94 60 33 86 58 86 76 82 91 24 6 58 80 6 78 11 48 12 38 4 82 23 75 8 49 94 86 75 56 86 49 92 42 22 96 2 95 52 57 62 66 74 56 7 17 10 63 39 36 27 7 47 59 94 50 59 75
85 39 39 81 47 1 99 95 58 84 84 28 16 69 60 92 79 75 43 70 73 41 29 17 76 1 52 100 87 4 44 26 87 24 46 38 23 37 100 16 53 96 64 18 34 34 17 31 22 46 24 8 97 67 37 14 16 55 60 65 90 72 43 25 39 61 39 24 85 65 23 1 69 48 98 27 38 64 4 59
11 45 36 29 60 77 51 19 14 94 9 71 15 47 29 16 82 86 46 5 17 17 92 86 40 72 75 37 25 1 85 18 99 57 35 35 49 72 79 13 14 8 57 81 39 55 93 96 10 44 23 99 75 58 56 84 77 61 6 71 26 23 12 2 84 21 3 22 89 23 34 59 88 55 67 43 97 26 64 7
25 46 91 2 60 79 90 61 57 86 31 78 73 73 41 5 40 97 82 52 43 34 67 24 76 74 47 30 77 51 72 92 54 16 48 32 38 24 54 28 90 71 59 36 10 47 13 10 34 1 47 39 71 9 98 95 11 2 46 45 12 91 43 8 77 27 52 19 31 33 76 90 98 41 88 47 78 10 83 78
99 77 2 70 77 93 4 49 64 93 34 92 73 52 16 56 19 17 79 86 71 55 93 52 96 83 16 94 76 73 75 59 50 22 7 41 37 73 63 20 35 93 89 63 100 80 55 21 91 52 63 82 52 81 28 95 89 17 26 6 76 24 45 80 63 63 23 92 92 5 53 27 85 21 75 22 25 78 49 74
5 67 2 21 60 59 93 18 19 19 23 53 83 27 10 73 10 72 69 40 9 73 14 6 25 89 66 46 89 31 69 19 5 63 31 81 26 66 4 94 76 9 65 53 31 26 37 74 75 74 49 36 36 66 72 53 11 94 96 78 6 17 99 33 20 6 7 77 31 32 90 73 47 80 94 33 50 2 94 8
95 8 56 68 22 14 17 5 31 75 8 22 58 31 51 23 49 98 12 5 13 99 22 1 45 15 65 78 15 2 57 62 81 2 92 52 1 7 96 1 73 76 58 27 13 45 66 85 100 14 88 62 98 47 44 73 96 22 43 42 15 54 91 84 1 92 99 61 87 77 1 21 82 57 23 12 65 67 46 55
49 83 22 63 25 86 84 86 74 66 98 43 76 87 79 6 75 91 4 46 80 78 9 34 69 79 21 65 76 50 41 49 64 36 70 92 59 90 47 58 62 37 52 36 100 20 41 37 16 93 84 23 26 100 74 35 90 1 35 86 9 33 95 28 26 92 21 77 26 94 47 39 85 77 92 58 22 69 78 12
55 45 58 85 7 2 70 76 30 95 38 79 88 32 78 35 74 86 26 95 12 82 75 48 25 86 79 23 71 38 88 87 28 16 14 82 46 1 5 49 48 57 47 25 74 91 76 95 77 91 33 52 64 66 52 69 84 46 71 53 33 91 61 34 52 67 68 76 75 19 17 34 68 68 63 23 26 51 48 82
73 97 78 30 74 83 60 3 65 47 74 99 18 17 100 6 5 39 4 82 46 42 28 100 6 79 13 37 72 27 98 70 52 83 10 57 79 68 73 45 41 52 87 30 78 99 74 90 5 15 23 17 64 45 25 54 10 6 57 44 40 55 55 62 33 33 12 55 10 61 6 75 42 89 19 24 79 53 78 6
19 31 4 60 76 32 32 64 17 29 4 59 95 67 2 22 44 91 68 62 3 23 76 81 62 26 20 71 63 32 79 94 76 56 95 55 94 36 41 15 58 13 53 58 83 52 67 99 94 98 34 89 82 40 53 12 88 48 96 18 41 83 32 3 35 29 65 14 23 72 57 5 61 33 52 85 38 69 94 23
34 5 55 60 92 70 55 23 25 38 62 66 57 2 34 96 20 38 16 1 38 36 59 36 27 69 89 72 32 2 52 16 20 68 97 52 80 88 16 14 27 4 31 21 3 97 3 81 87 27 62 33 87 56 15 59 23 69 16 16 11 83 79 82 44 95 31 92 50 75 68 5 52 51 10 37 100 60 34 26
12 97 54 65 26 25 50 51 85 10 74 12 81 81 37 7 51 7 78 28 86 18 45 5 39 18 14 45 23 32 19 80 88 70 65 49 11 94 55 27 17 3 90 44 5 67 2 13 24 43 43 60 97 69 21 93 21 85 66 49 96 16 74 30 34 77 70 61 27 87 2 55 35 96 20 77 63 41 80 39
43 15 15 9 34 92 47 97 4 56 3 82 59 40 52 44 38 39 73 26 37 44 63 3 40 33 9 2 80 75 49 25 16 40 9 69 1 35 68 21 30 7 43 47 57 9 68 7 76 25 50 15 61 90 50 23 7 91 100 78 91 94 16 92 99 100 30 3 2 55 74 98 42 39 80 48 13 58 62 29
9 56 61 92 48 43 96 56 24 56 25 56 20 37 93 75 85 37 12 96 70 71 3 18 61 27 65 65 87 3 18 60 88 7 59 26 92 41 76 29 68 23 64 77 7 84 41 26 57 50 25 81 28 75 18 15 61 91 23 55 91 92 89 35 45 22 27 75 23 80 14 54 66 49 10 63 48 15 59 77
96 56 44 3 16 71 56 89 100 72 36 53 2 2 50 11 65 2 93 81 25 78 31 56 98 52 49 97 48 59 87 84 80 25 65 53 14 40 91 10 13 28 3 11 31 71 13 98 58 78 49 99 56 97 45 73 92 54 75 51 18 79 93 12 43 95 37 68 79 13 5 80 77 93 64 97 90 45 75 42
50 6 100 18 74 36 46 35 12 58 52 6 78 15 42 40 56 85 99 66 93 86 73 10 22 79 91 68 95 32 98 74 12 30 9 56 66 20 42 73 27 21 42 12 13 99 12 73 75 92 19 49 43 64 44 61 74 32 61 51 67 14 88 83 5 39 60 51 28 15 31 72 28 12 27 19 30 40 50 85
25 19 25 16 46 21 52 87 48 54 49 74 29 69 84 87 60 75 82 43 56 79 91 27 59 8 34 19 29 89 57 54 1 93 54 19 67 55 15 6 69 44 11 45 65 76 3 72 61 68 6 78 51 59 5 42 1 16 23 50 54 90 62 83 11 57 17 83 87 50 52 69 43 22 11 9 8 55 40 9
88 82 23 70 32 39 5 37 93 17 7 38 98 91 46 4 55 20 78 51 69 52 76 33 88 45 94 1 68 11 98 39 41 91 88 18 99 4 16 24 83 29 20 74 83 45 8 70 90 4 1 83 86 99 13 64 55 91 35 77 15 54 76 41 53 73 62 26 21 77 16 8 7 18 20 18 51 51 85 55
43 56 87 71 48 10 13 24 77 14 74 97 92 94 63 70 35 39 23 14 88 2 43 56 41 83 59 7 58 17 19 69 38 40 83 12 29 17 19 60 36 73 11 54 16 47 25 73 51 93 46 35 18 79 69 56 70 57 19 99 54 75 44 84 75 100 57 64 16 76 36 40 2 33 83 15 73 51 49 71
33 84 19 60 95 5 64 89 57 18 50 88 93 74 91 46 35 89 81 49 54 32 54 89 34 61 49 8 1 48 75 54 54 59 50 41 41 2 58 57 14 12 99 54 66 25 24 25 20 70 52 95 93 16 71 42 45 69 53 49 67 21 91 69 49 6 32 46 86 61 6 70 94 62 34 55 93 68 28 16
76 59 88 32 87 40 34 87 73 69 95 87 36 30 14 88 34 85 80 76 77 7 76 12 76 57 88 5 71 75 42 95 87 28 59 42 15 45 43 32 8 49 60 45 43 32 15 51 34 57 38 50 62 57 81 33 56 23 28 83 35 43 54 31 91 62 73 96 33 37 67 98 86 82 19 79 99 45 82 71
67 96 59 44 98 16 19 35 54 42 66 29 37 71 44 91 82 88 56 18 82 38 21 44 89 59 80 1 11 99 3 47 43 20 45 75 19 87 24 45 86 45 66 76 81 87 28 51 71 58 90 85 69 75 93 14 63 36 95 12 32 66 19 91 6 49 10 65 71 71 99 7 93 68 96 30 56 47 91 83
86 88 10 76 22 57 55 100 94 38 45 11 64 64 13 15 42 33 46 60 77 70 85 73 74 52 33 65 76 92 40 51 32 15 69 87 29 11 36 55 97 55 37 14 98 63 17 92 85 38 19 79 30 18 41 80 53 52 4 89 95 45 92 44 48 26 89 38 31 73 7 97 61 49 35 57 30 86 80 97
18 83 79 31 37 38 99 3 79 84 65 40 33 70 77 25 62 21 88 42 31 99 52 61 44 76 47 61 20 81 20 30 57 56 41 87 18 6 63 98 23 24 41 36 56 15 13 35 78 73 17 38 29 93 47 72 92 70 10 69 33 1 95 95 46 50 69 79 21 7 21 50 45 13 23 28 22 73 43 32
93 12 40 66 45 72 81 31 56 61 63 81 8 16 64 61 10 88 63 43 94 84 94 25 65 46 99 56 82 71 99 86 90 44 42 2 64 55 45 55 13 16 2 17 13 100 100 39 44 22 80 7 34 73 97 53 15 59 93 85 84 97 54 100 97 72 58 90 97 78 51 15 25 51 99 18 52 53 33 25
48 32 55 32 56 58 54 74 92 36 94 21 49 3 55 74 10 25 56 27 28 9 59 84 54 83 59 47 90 17 17 63 70 35 52 57 80 28 74 78 95 6 8 70 44 42 46 41 54 85 100 56 73 13 6 80 39 38 93 80 10 6 64 82 7 33 28 14 11 98 48 7 92 4 1 92 89 85 11 79
83 86 51 26 53 61 43 49 81 26 17 89 92 51 16 1 50 44 41 27 59 100 76 48 84 34 9 73 64 88 35 30 23 60 48 59 19 9 58 6 34 31 76 43 16 4 35 12 47 8 57 33 82 54 74 81 74 64 58 71 41 8 92 50 54 49 99 33 71 54 27 44 16 47 40 25 47 2 96 7
39 7 4 67 60 49 18 34 58 35 70 49 53 79 60 37 66 7 28 75 13 90 39 97 34 48 91 19 76 60 15 83 40 39 36 41 99 3 41 44 32 46 92 79 95 91 35 9 77 45 24 65 62 79 65 83 25 53 44 1 83 86 36 67 40 63 75 64 41 22 26 22 44 61 92 33 20 51 47 17
78 99 49 40 8 77 29 21 83 58 64 100 49 28 68 27 87 39 40 26 9 34 13 4 59 36 87 28 58 32 66 73 1 81 18 83 6 66 71 51 86 15 93 11 92 78 42 64 3 96 87 73 67 57 83 95 64 72 83 51 55 68 11 94 87 76 64 18 73 8 28 54 13 81 93 62 78 77 42 76
86 37 44 44 55 97 42 73 55 96 2 94 70 1 6 91 91 63 64 61 14 42 15 53 25 75 83 53 70 84 80 25 83 90 88 32 62 93 64 76 1 64 60 90 63 30 76 44 37 94 39 17 22 76 68 16 96 33 96 5 41 31 5 94 30 15 94 7 95 18 81 76 1 68 59 98 79 56 44 49
93 93 64 88 4 38 91 47 72 88 42 97 85 15 32 40 99 17 70 19 38 2 99 59 87 9 57 30 56 13 46 98 92 9 24 80 29 37 97 27 60 47 59 69 84 3 73 5 67 30 45 24 76 11 8 99 43 6 25 58 60 99 10 96 33 77 29 5 19 24 41 31 5 98 38 69 89 63 29 87
89 40 49 20 71 48 31 95 21 7 19 62 83 61 5 22 32 21 17 33 84 23 31 91 57 73 56 70 9 3 86 65 89 47 20 37 37 13 24 63 50 16 28 59 99 72 53 89 53 9 83 86 91 83 67 11 94 8 79 11 14 67 50 40 23 13 90 86 6 23 32 39 81 2 87 44 54 38 16 71
41 6 83 7 93 94 18 28 23 8 29 56 50 32 40 54 28 8 25 39 28 17 73 16 34 39 64 69 87 73 54 34 66 43 18 29 60 10 79 73 65 92 93 73 68 25 61 5 63 11 39 32 69 63 65 97 46 6 23 63 10 32 41 65 34 92 7 58 30 26 38 6 2 36 20 55 13 82 23 52
96 50 19 42 68 61 76 78 16 20 60 49 36 12 43 44 85 18 86 100 74 59 95 51 4 93 16 41 100 64 100 24 48 86 41 8 29 16 64 80 4 22 52 15 88 79 7 88 9 39 63 70 49 64 67 45 78 29 7 55 32 86 64 84 1 26 77 10 89 59 3 37 58 81 6 77 50 91 25 31
64 13 71 44 81 33 52 22 73 58 28 77 72 40 1 39 69 26 72 28 22 50 96 99 50 72 30 87 14 91 98 11 89 27 33 99 68 14 81 73 79 84 56 36 77 46 33 96 32 86 89 59 62 45 83 90 33 84 5 78 53 60 97 12 76 98 97 6 21 35 11 15 78 81 10 15 73 16 31 44
80 55 4 26 60 37 88 28 54 7 74 9 77 72 90 12 74 80 75 89 84 89 96 85 77 31 21 22 63 7 91 77 83 91 62 57 81 84 28 51 41 19 41 28 48 9 1 77 42 63 52 32 18 59 90 84 69 5 60 13 71 86 73 86 20 72 69 33 68 16 66 74 79 35 14 28 38 46 31 6
80 7 43 88 58 13 4 67 36 59 76 45 41 29 38 43 27 30 19 24 5 85 70 97 71 68 5 4 33 24 70 47 68 46 40 39 96 47 71 63 96 13 99 37 70 85 25 4 40 39 55 4 72 45 53 64 97 36 67 4 47 81 1 61 95 86 75 91 8 46 9 70 88 44 83 45 85 1 62 82
4 25 94 64 61 26 2 48 65 85 31 88 31 79 57 71 20 86 84 92 67 81 80 58 85 19 55 21 66 73 81 52 98 75 3 85 93 99 49 72 47 82 2 56 81 89 63 1 31 34 84 8 12 48 2 6 29 74 60 98 1 25 87 6 34 85 22 46 47 81 70 40 28 24 98 77 10 68 20 75
28 50 70 65 11 32 87 99 66 42 36 43 71 70 75 8 56 60 15 79 51 34 1 69 39 1 85 46 33 70 39 89 60 91 79 78 4 23 97 11 47 42 66 44 19 98 13 99 41 95 38 84 50 44 55 6 12 99 86 39 54 33 57 68 81 68 5 23 8 27 14 28 89 34 86 17 53 27 10 94
64 98 44 1 50 58 75 10 96 29 89 49 97 60 96 67 45 61 30 77 74 92 57 93 78 14 12 91 93 69 84 25 69 65 88 3 77 24 97 12 46 52 27 7 6 76 90 47 32 30 97 75 86 76 99 62 40 2 21 37 36 62 21 16 77 17 94 89 91 87 94 65 78 74 94 13 70 55 54 54
40 16 91 37 60 1 66 36 6 15 76 59 97 20 100 64 78 36 11 23 42 34 45 99 68 13 52 83 36 3 40 54 83 10 6 67 89 62 48 54 37 40 73 41 88 34 14 72 30 84 9 25 52 61 22 65 64 44 13 61 85 59 97 98 68 56 60 18 51 75 76 48 19 69 56 100 10 44 85 28
69 21 34 20 92 12 93 55 87 15 15 36 35 96 32 78 95 14 88 48 77 88 83 2 58 55 62 58 93 11 7 72 61 33 85 59 2 56 42 73 36 43 100 23 20 12 9 59 25 85 46 40 80 74 31 65 17 92 98 74 35 97 71 85 4 90 75 86 38 43 82 92 55 99 95 26 91 97 68 55
77 52 23 27 22 80 46 80 69 74 76 53 91 52 18 70 73 91 39 58 81 4 80 87 17 15 52 50 4 62 65 64 87 51 78 14 93 74 43 83 76 77 96 53 66 4 35 63 8 32 33 29 46 20 60 42 61 63 88 13 68 80 53 58 67 19 91 70 43 29 39 30 100 51 23 12 38 67 37 88
77 10 45 37 1 56 67 46 27 52 70 37 21 65 59 47 94 70 62 50 28 59 24 54 4 34 72 77 29 63 71 98 69 36 31 41 35 93 15 58 23 37 74 98 2 15 60 37 71 78 4 67 18 2 52 26 69 41 67 69 42 15 50 16 51 54 88 34 75 22 66 94 97 18 80 46 72 6 22 99
50 78 72 79 99 41 18 42 85 76 60 98 26 27 68 42 11 72 27 18 36 48 48 82 2 83 60 31 64 56 87 5 19 7 58 60 86 57 15 58 54 76 80 62 15 40 24 65 38 67 12 55 25 77 87 58 81 85 67 48 31 39 14 52 31 75 22 97 8 33 77 35 23 23 66 100 83 48 70 95
30 45 68 25 75 95 81 39 78 31 56 45 36 69 59 3 72 84 57 98 55 81 70 91 13 79 62 41 18 53 10 50 61 44 53 50 16 33 17 72 6 29 83 7 63 97 19 6 52 41 73 52 29 73 89 40 58 91 76 31 98 67 25 13 46 68 81 68 83 51 43 61 84 63 53 60 97 64 33 85
43 31 12 26 45 17 87 42 44 56 57 8 85 24 18 69 26 95 91 34 8 78 88 75 36 58 9 10 17 88 65 72 19 69 62 16 100 46 42 43 90 65 40 3 85 78 37 28 35 63 80 40 40 53 86 96 26 89 59 29 35 19 83 88 46 62 25 10 60 35 28 61 37 30 17 21 21 44 50 40
87 67 35 42 81 4 15 94 94 76 91 23 11 80 80 8 55 57 31 29 63 40 32 31 31 99 88 40 42 18 90 96 2 63 23 69 41 15 87 72 97 39 61 30 62 92 10 81 59 51 91 91 44 13 71 90 10 12 90 22 14 77 80 25 78 12 87 80 19 99 44 11 30 43 90 93 42 69 35 45
60 98 90 10 95 95 63 64 33 22 98 69 36 92 28 93 17 89 48 75 48 7 80 70 45 67 49 20 37 57 35 62 40 9 82 38 77 53 6 11 57 20 92 61 64 67 39 74 66 94 76 2 95 11 18 69 67 39 26 36 87 3 4 63 54 84 65 9 73 31 63 38 92 78 40 76 38 18 75 27
37 15 46 66 12 7 58 76 79 15 94 27 82 80 88 32 18 59 90 77 63 62 14 89 8 81 91 24 83 57 34 77 22 17 50 38 3 28 72 91 29 48 68 7 84 42 70 90 33 91 57 73 83 48 24 33 84 55 96 4 2 33 11 22 90 22 84 90 36 16 81 91 5 4 48 12 48 12 14 39
86 71 51 89 78 19 88 89 49 78 7 17 99 49 11 60 8 40 45 55 28 45 60 3 3 30 70 28 30 38 37 33 13 23 59 38 31 67 66 49 61 86 77 93 42 77 77 51 12 15 68 11 30 83 5 63 20 59 47 29 31 80 74 84 98 27 17 39 94 58 6 25 6 61 32 62 16 74 92 93
95 13 25 8 58 48 5 44 91 12 61 42 10 3 9 82 46 92 5 88 35 22 97 34 51 81 66 63 60 15 71 24 75 24 57 2 29 14 95 92 44 62 94 7 67 66 26 77 87 57 84 17 43 10 38 7 73 69 98 41 66 40 59 58 78 67 30 63 41 44 35 41 32 8 82 1 42 57 10 72
31 2 69 15 1 30 93 96 38 53 61 94 26 45 34 78 59 66 78 69 68 99 5 74 98 27 3 38 91 96 23 87 22 60 52 21 61 42 54 57 28 34 97 64 33 38 39 45 83 94 25 15 62 85 77 52 90 45 79 80 11 36 35 55 90 9 33 15 51 75 54 27 66 22 64 6 78 100 78 37
72 22 85 1 17 56 52 98 4 44 13 27 33 32 38 51 25 35 77 29 20 52 4 85 49 39 45 85 41 10 87 21 4 94 44 75 39 19 37 40 40 79 53 41 66 68 14 55 78 85 98 39 79 45 44 51 35 72 61 91 7 8 39 57 31 98 97 46 27 14 76 94 71 7 60 72 92 92 29 39
2 72 83 86 92 95 26 1 82 2 30 14 78 3 33 77 67 63 36 65 5 66 90 72 96 62 88 59 85 81 34 3 59 26 89 89 3 52 7 4 21 44 42 33 96 14 9 50 74 27 45 64 81 74 74 27 3 50 27 10 24 54 37 92 63 25 53 21 9 94 26 26 69 38 52 29 12 49 47 39
95 68 37 51 80 50 4 78 77 78 5 26 5 59 11 1 57 80 36 72 92 40 83 86 32 52 57 40 63 44 32 4 38 78 19 51 53 86 20 45 1 95 80 65 85 28 15 51 97 72 53 92 66 27 98 5 16 42 63 28 3 39 19 97 15 19 47 55 24 47 22 9 11 43 26 73 3 46 17 100
68 44 97 64 67 34 72 48 48 45 64 60 90 95 68 15 71 13 42 35 18 8 63 84 39 92 26 18 59 32 81 74 33 15 82 20 17 37 16 72 97 32 51 33 99 3 100 41 9 7 58 8 98 5 66 32 41 1 72 22 13 78 20 92 32 24 6 97 62 40 38 80 53 67 5 9 95 35 25 71
9 81 65 28 24 30 43 5 90 19 53 55 37 40 64 22 9 12 24 38 69 67 2 41 90 67 20 98 100 52 49 17 56 48 79 5 28 12 7 85 21 24 96 52 84 83 84 57 97 13 25 35 39 79 45 49 19 35 90 55 76 1 27 24 20 43 58 26 62 69 36 86 90 14 61 5 49 90 49 63
46 66 87 12 90 99 74 18 85 78 14 61 7 94 64 51 87 70 28 12 91 87 12 51 6 52 8 78 8 25 28 62 34 83 16 38 100 87 89 91 46 51 13 14 63 47 58 74 28 9 32 42 66 6 23 80 84 90 37 24 86 88 90 26 45 12 43 95 16 11 94 17 80 67 90 52 54 72 79 81
8 52 50 7 55 56 83 29 40 37 65 42 22 36 74 77 81 6 32 9 83 76 54 2 58 88 28 86 19 58 87 17 93 44 10 10 83 15 6 11 38 68 87 64 84 96 57 33 96 84 92 72 91 21 96 56 17 49 46 97 98 40 40 41 5 52 41 68 12 14 15 13 79 27 83 43 80 92 9 33
13 74 22 30 26 26 84 58 23 38 65 67 55 79 48 49 16 93 38 55 24 54 78 31 41 86 90 34 47 72 56 29 53 48 22 65 47 48 78 72 10 68 67 78 3 57 26 34 8 21 69 58 51 97 28 82 74 6 95 53 39 85 61 98 2 51 32 72 94 1 59 23 53 61 34 73 29 28 46 90
31 10 71 57 11 25 75 80 22 34 38 84 34 46 68 43 50 34 1 96 16 99 70 26 93 19 77 5 17 11 24 7 28 11 54 52 74 29 5 23 59 10 95 63 20 15 58 48 8 45 72 91 72 2 23 25 52 50 65 56 11 98 53 24 23 76 92 88 99 77 82 63 75 50 83 87 23 28 46 5
100 69 85 59 22 44 58 72 92 64 100 100 44 70 20 2 12 30 89 52 82 70 12 34 57 67 92 39 37 14 88 34 87 76 80 91 42 67 84 12 92 87 13 44 23 45 1 91 88 16 67 84 55 35 49 50 69 92 88 95 53 24 12 96 56 56 22 67 65 50 19 16 49 70 64 69 68 69 52 61
82 54 28 52 63 88 16 10 76 81 51 48 31 78 86 66 91 76 37 16 14 81 78 24 33 65 25 33 3 22 86 39 58 32 99 77 97 17 53 18 24 98 55 89 100 44 23 68 72 53 27 73 29 32 34 12 82 100 21 34 61 34 26 62 89 2 68 82 5 19 56 70 99 4 34 86 47 7 50 63
40 5 24 14 2 44 5 85 63 88 43 67 33 67 60 4 85 13 71 87 40 44 18 19 13 50 27 7 68 66 4 54 29 42 18 7 94 36 21 10 17 72 19 65 11 40 93 39 52 20 69 21 100 67 14 88 20 8 82 40 43 65 35 49 77 83 87 81 12 44 72 43 6 48 49 58 67 93 23 10
49 64 81 46 3 97 36 81 62 86 60 74 13 20 45 80 4 89 54 58 80 53 70 32 52 61 82 16 10 79 44 90 9 91 15 32 5 47 97 99 66 49 95 8 88 84 53 12 84 18 56 76 58 89 69 74 62 50 65 30 47 32 35 85 46 19 8 55 48 78 100 48 40 32 29 90 94 32 75 91
79 40 99 23 27 14 91 59 85 27 62 89 2 34 33 17 86 19 82 61 6 77 65 30 65 59 46 33 55 8 44 87 48 19 21 69 46 55 91 21 67 48 65 35 76 37 80 78 89 83 39 84 10 17 56 89 75 86 32 76 25 28 47 28 36 73 41 32 11 29 26 88 78 13 79 34 71 63 86 22
89 88 42 99 96 28 97 39 84 78 81 83 98 73 4 92 25 8 47 3 78 87 74 98 49 71 55 69 89 6 27 15 80 29 44 14 52 4 85 18 44 81 84 82 29 62 3 49 73 75 84 62 96 25 45 13 9 13 45 9 41 41 5 67 13 86 32 25 19 36 95 97 13 78 1 29 2 95 85 62
43 74 43 2 51 3 70 44 52 95 1 72 16 8 34 77 80 39 90 61 52 92 71 16 33 90 5 53 50 48 100 35 59 9 52 46 86 33 62 80 84 50 84 56 56 84 1 74 33 80 18 49 55 5 18 47 27 90 91 21 16 10 46 57 90 53 7 98 38 72 95 3 16 68 69 12 27 27 58 52
89
10 71 25 45
34 47 35 51
23 59 77 80
36 53 71 79
29 37 50 62
47 69 23 55
47 60 32 34
11 49 30 58
16 72 17 57
11 45 3 76
4 56 46 64
38 58 1 58
1 45 47 71
58 66 15 19
19 27 45 64
40 47 8 10
31 58 50 77
36 60 19 26
14 16 7 44
80 84 27 61
33 63 3 8
29 46 7 80
70 84 66 66
28 48 50 56
33 41 43 43
70 79 68 73
39 48 56 78
3 7 53 55
6 77 33 38
13 60 64 73
18 19 15 43
69 69 13 78
44 53 76 80
8 43 20 75
73 77 17 58
54 72 64 64
71 84 38 75
29 41 41 73
18 54 52 59
26 32 40 72
47 50 38 73
6 25 53 78
35 76 50 69
37 70 11 49
48 58 35 69
53 84 34 63
58 69 2 52
83 84 67 72
6 6 39 71
16 30 71 80
38 45 76 79
20 74 54 64
13 42 73 79
50 57 59 73
42 59 8 57
69 69 30 65
6 44 1 62
75 77 24 74
20 27 14 17
24 64 50 64
67 77 55 56
34 43 69 78
63 83 54 67
79 80 42 49
2 23 51 74
21 37 42 46
67 84 40 40
40 46 46 52
65 78 30 57
51 67 79 79
36 76 60 69
61 66 22 26
70 80 46 80
84 84 44 61
66 71 75 75
84 84 58 77
35 72 28 62
45 55 41 51
53 61 11 22
75 79 50 61
14 76 65 79
8 76 30 74
74 83 21 64
13 17 6 74
18 26 35 80
18 19 51 79
79 81 18 20
9 55 78 80
66 73 43 58
