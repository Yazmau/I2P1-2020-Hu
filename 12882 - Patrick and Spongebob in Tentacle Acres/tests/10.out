100
100
100
99
99
100
98
100
100
100
100
100
100
95
100
96
100
100
100
100
100
100
86
100
99
99
100
93
100
100
97
98
98
100
100
98
100
100
100
100
99
100
100
100
100
100
100
98
98
100
98
100
100
98
100
98
100
99
100
100
98
99
100
100
100
100
99
100
100
92
100
99
100
91
82
98
100
99
100
100
100
100
100
100
100
98
89
97
100
